Add optional txids param to listspendtxs (#921)
wizardsardine/liana: doc/API.md

List stored Spend transactions.

> If `txids` specified, only transactions with an id matching will be retrieved.

Reviewer: ```suggestion
If `txids` is specified, only list transactions whose `txid` is in `txids`.
```

#### Request

| Field         | Type                       | Description                           |
| ------------- | -------------------------- | ------------------------------------- |

> | `txids`       | array of string (optional) | Ids of the transactions  to retrieve  |

Reviewer: nit
```suggestion
| `txids`       | array of string (optional) | Ids of the transactions to retrieve  |
```

Author: fixed
